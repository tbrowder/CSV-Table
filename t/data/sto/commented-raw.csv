# a commen,age
Sally Je# 
      name,age
# Sally is my sister
Sally Jean,# she really is
# another comment
# 
