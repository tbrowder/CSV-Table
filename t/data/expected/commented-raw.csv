# a commented CSV file
# 
name, age
# Sally is my sister
Sally   Jean,21 # she really is
  # another comment
# 
